api/src/main/java/org/openmrs/module/fhir2/api/translators/ConceptTranslator.java

```diff
@@ -25,6 +28,15 @@ public interface ConceptTranslator extends OpenmrsFhirTranslator<Concept, Codeab
 	@Override
 	CodeableConcept toFhirResource(@Nonnull Concept concept);
 	
+	/**
+	 * Maps a collection of {@link org.openmrs.Concept}s to a
+	 * {@link org.hl7.fhir.r4.model.CodeableConcept}
+	 *
+	 * @param openmrsConcepts the collection of OpenMRS concepts to translate
+	 * @return the mapping of OpenMRS concept to corresponding FHIR concept resource
+	 */
+	List<CodeableConcept> toFhirResources(Collection<Concept> openmrsConcepts);
```

Comment:
Can we add a since annotation to this? https://openmrs.atlassian.net/wiki/spaces/docs/pages/25477044/Java+Conventions#New-Public-Methods-and-Classes

Reply:
I added this annotation to newly added interface methods